clients/java/signalr/build.gradle

```diff
@@ -20,6 +21,22 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.8.5'
 }
 
+spotless {
+    java {
+        licenseHeader '// Copyright (c) .NET Foundation. All rights reserved.\n' +
+        '// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.\n' // License header
+
+        importOrder 'java', 'javax', 'org', 'com', 'com.diffplug', ''	// A sequence of package names
+
+        replace      'Not enough space after if', 'if(', 'if ('
```

Comment:
for loops and while loops too

Reply:
This is my concern about custom rules... 😟

Reply:
And `switch`...